tests/test_urlslug.py

```diff
@@ -188,3 +188,24 @@ def test_removes_apostrophes(self):
 
         expected = '9-4-ohms-law'
         assert expected == slug
+
+    def test_handling_unicode(self):
+        titles = (
+            b'College Physics',
+            u'<span class="os-number">5</span><span class="os-divider"> </span><span class="os-text">'
+            u'Further Applications of Newton’s Laws: Friction, Drag, and Elasticity</span>'.encode('utf-8'),
+            u'<span class="os-text">Introduction: Further Applications of Newton’s Laws</span>'.encode('utf-8')
+        )
+
+        slug = generate_slug(*titles)
+        assert slug == '5-introduction-further-applications-of-newtons-laws'
+
+        titles = (
+            u'College Physics',
+            u'<span class="os-number">5</span><span class="os-divider"> </span><span class="os-text">'
+            u'Further Applications of Newton’s Laws: Friction, Drag, and Elasticity</span>',
+            u'<span class="os-text">Introduction: Further Applications of Newton’s Laws</span>',
+        )
```

Comment:
Can you split this test into two distinct test functions? And also say that this second one is specifically test the handling of unicode in the parent title?

Reply:
Actually the two tests have the same text, just one has the inputs as bytes and the other one as unicode.  I'll split them into different functions.